fhir-persistence-jdbc/src/test/java/com/ibm/fhir/persistence/jdbc/test/FHIRDbDAOTest.java

```diff
@@ -52,9 +69,14 @@ public void testGetDB2Connection() throws Exception {
         props.setProperty(FHIRDbDAO.PROPERTY_DB_URL, "jdbc:db2://localhost:50000/fhirdb");
         props.setProperty(FHIRDbDAO.PROPERTY_DB2_USER, "user");
         props.setProperty(FHIRDbDAO.PROPERTY_DB2_PSWD, "password");
```

Comment:
use the constants to be consistent

Reply:
Yes, should be part of the FHIRDbDAO properties cleanup.

Reply:
removed because class no longer exists.